biomage/utils/data.py

```diff
@@ -0,0 +1,162 @@
+import boto3
+import click
+from botocore.exceptions import ClientError
+
+
+def modified_records(item, target_table, config):
+    """
+    Return modified records.
+    This function should return spreadable dictionary
+    """
+
+    if target_table == config["staging-experiments-table"]:
+        return {
+            # Rewrite pipeline details in experiments-table
+            # metadata to pipeline ARN in production environment
+            "meta": {
+                "M": {
+                    **item["meta"]["M"],
+                    "pipeline": {
+                        "M": {
+                            "stateMachineArn": {"S": ""},
+                            "executionArn": {"S": ""},
+                        }
+                    },
+                }
+            }
+        }
```

Comment:
I actually have no idea, I just moved agi's code to another place but haven't done any of that myself. @aerlaut what do you think?

Reply:
Maybe what we could do is instead of hardcoding the schema format of `meta`, we read it from what the schema is in production. Then return it instead of this hardcoded dictionary. Would that work?

Reply:
I don't think it is too dangerous if we change the schema and break this: these are transient values that are overwritten every time that the pipeline is started.

Reply:
as I mentioned I am unsure about this because agi did it. I would merge it already and then when/if it breaks we can revisit it. I don't think it's urgent. What do you think @ivababukova ?

Reply:
yes, let's merge this.

Reply:
it's not urgent